dex-it-common/src/main/scala/com/wavesplatform/dex/it/api/dex/AsyncEnrichedDexApi.scala

```diff
@@ -479,6 +479,14 @@ class AsyncEnrichedDexApi(apiKey: String, host: => InetSocketAddress)(implicit e
 
   override def getMatcherConfig: R[Config] = getMatcherConfig(apiKeyHeaders)
 
+  override def getAddressActorState(address: String, headers: Map[String, String]): R[HttpAddressActorState] = mk {
```

Comment:
We should hide an implementation details in API, because we should be able to change implementation and don't affect other places. Let's rename these methods from `getAddressActorState` to `getAddressState`

Reply:
done